static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -211,46 +211,80 @@ a[href].contributions__contribute.contributions__contribute--epic,
     }
 }
 
-.contributions__adblock--moment {
+.contributions__adblock {
     position: absolute;
     top: 0;
     padding-top: $gs-baseline/2;
+    width: 100%;
 
-    .contributions__adblock--moment-content {
-        border-top: 5px solid #052962;
-        background-color: #f6f6f6;
+    .contributions__adblock-content {
+        border-top: 5px solid $highlight-main;
+        margin-right: 20px;
     }
 
-    .contributions__adblock--moment-header {
-        padding-left: 13px;
+    .contributions__adblock-header {
+        padding-left: 10px;
+        padding-top: 3px;
 
         h2 {
```

Comment:
this clearly isn't new, but I'm not sure if relying on nesting is best practice for BEM or not?

Reply:
yeah agreed, this could be unnested especially since it's all namespaced with `adblock-`